### PR TITLE
devmessias gsoc posts  part 2: weeks 09, 10 and 11

### PR DESCRIPTION
**Pull request by devmessias**

### CODE REVIEW

**Review by skoudoro — COMMENTED** [review]
Hi @devmessias,

Thank you for this PR. Do not forget to add the blog headers on each post. See below the error I got:

```
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-06-28-gsoc-devmessias-4.rst:65: WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 6 supplied.

.. code-block:: python
   from abc import ABC, abstractmethod

   GenericImageBufferManager(ABC):
       def __init__(
               self, max_window_size=None, num_buffers=2, use_shared_mem=False):
            ...
       @abstractmethod
       def load_mem_resource(self):
           pass
       @abstractmethod
       def create_mem_resource(self):
           pass
       @abstractmethod
       def cleanup(self):
           pass
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-07-05-gsoc-devmessias-5.rst:35: WARNING: Bullet list ends without a blank line; unexpected unindent.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-07-05-gsoc-devmessias-5.rst:38: WARNING: Bullet list ends without a blank line; unexpected unindent.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-07-05-gsoc-devmessias-5.rst:43: WARNING: Bullet list ends without a blank line; unexpected unindent.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-07-26-gsoc-devmessias-8.rst:19: WARNING: Bullet list ends without a blank line; unexpected unindent.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:335: WARNING: Unexpected indentation.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:334: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/koudoro/Software/fury/docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:337: WARNING: Block quote ends without a blank line; unexpected unindent.
```

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst [hunk 1]
```diff
@@ -0,0 +1,406 @@
+Google Summer of Code 2021 Final Work Product
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:1]: Why it appears a second time below?

File: docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Hi everyone! My name is Bruno Messias. Currently I'm a Ph.D student at
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst:1]: Can you add a header? Which week is it?

File: docs/source/posts/2021/2021-23-08-gsoc-devmessias-final-report.rst [hunk 3]
```diff
@@ -0,0 +1,404 @@
+Google Summer of Code 2021 Final Work Product
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-23-08-gsoc-devmessias-final-report.rst:1]: same comment as above

**Review by skoudoro — COMMENTED** [review]
Hi @devmessias,

Can you finalize this? The new GSOC season will start and we would like all the previous session reports.
- Some headers are missing
- final report is duplicated
- Seems to miss some weeks, I see only the 11, missing 9-10

Thanks

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst (hunk 1 above)
Comment (devmessias) [reply docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:1]: Ok,  thanks for pointing this out. I'll take look.

File: docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
+Week #11: Welcome to my weekly Blogs!
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst:1]: `#10` instead

File: docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
+Week #11: Welcome to my weekly Blogs!
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst:1]: `#9` instead

File: docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+====================================
+
+.. post:: August 16 2021
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst:4]: August 2 instead

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst [hunk 7]
```diff
@@ -0,0 +1,422 @@
...
+--------------------
+
+-  .. rubric:: Create a streaming system (stadia-like) for FURY
+      :name: create-a-streaming-system-stadia-like-for-fury
+
+   There are several reasons to have a streaming system for data
+   visualization. Because I am doing my Ph.D. in a developing country I
+   always need to think of the less expensive solutions to use the
+   computational resources available. For example, with the GPU’s prices
+   increasing, it is necessary to share the a single machine with GPU
```
[the remark this answers is not in the document]
Comment (skoudoro) [reply docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:57]: typo, see above

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst [hunk 8]
```diff
@@ -0,0 +1,406 @@
...
+Below is presented the Helios Logo which was developed by my mentor
+Filipi Nascimento.
+
+.. raw:: html
+
+   <center>
+
+.. raw:: html
+
+   </center>
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:242]: I suppose, it is missing a image here

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst [hunk 9]
```diff
@@ -0,0 +1,406 @@
...
+   -  https://github.com/fury-gl/fury/pull/489
+
+      .. raw:: html
+
+         <center>
+
+      .. raw:: html
+
+         </center>
+
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:279]: same comment. Missing an image or what is the goal

File: docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+====================================
+
+.. post:: August 16 2021
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst:4]: August 9 instead

**Review by skoudoro — COMMENTED** [review]
Thank you for the update @devmessias.

See below for some quick fixes. I will generate the page now to see if the rendering is ok.

File: docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst (hunk 4 above)
Comment (skoudoro) [reply docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst:1]: could you also update the title?

File: docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst (hunk 5 above)
Comment (skoudoro) [reply docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst:1]: could you also update the title?

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst (hunk 1 above)
Comment (skoudoro) [reply docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:1]: could you add the header in the final report too?

File: docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst [hunk 11]
```diff
@@ -0,0 +1,391 @@
+Google Summer of Code 2021 Final Work Product
+=============================================
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-gsoc-devmessias-final-report.rst:2]: could you add the header in the final report too?

File: docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst [hunk 12]
```diff
@@ -0,0 +1,46 @@
+Week #09: Welcome to my weekly Blogs!
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-02-gsoc-devmessias-9.rst:1]: Can you update the title? you have 3 times the same

File: docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst [hunk 13]
```diff
@@ -0,0 +1,52 @@
+Week #10: Welcome to my weekly Blogs!
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-09-08-gsoc-devmessias-10.rst:1]: Can you update the title? you have 3 times the same

File: docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst [hunk 14]
```diff
@@ -0,0 +1,99 @@
+Week #11: Welcome to my weekly Blogs!
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst:1]: Can you update the title? you have 3 times the same

File: docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+   *   
+
+       The first example, viz_huge_amount_of_labels.py, shows that feature has a realy good performance. The user can 
+       draw hundreds of thounsands of characters in a regular computer.
+
+     ![](https://user-images.githubusercontent.com/6979335/129643743-6cb12c06-3415-4a02-ba43-ccc97003b02d.png)
+
+   *   The second example, viz_billboad_labels.py, shows the different behaviors of the label actor. In addition, presents 
+     to the user how to create a new texture atlas font to be used across different visualizations.
+
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst:39]: Styling issue when you generate the documents

File: docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+   -  .. rubric:: Flickering
+         :name: flickering
+
+      ::
+
+           Finally, I could found the cause of the flickering effect on the streaming system. 
+           This flickering was appearing only when the streaming was created using the Widget object. 
+           The cause seems to be a bug or a strange behavior from vtk. 
+           Calling  
+
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-16-08-gsoc-devmessias-11.rst:63]: styling issue when you generate the document

**Comment by skoudoro** [comment]
Thank you for all those updates @devmessias. This is ready to go. Merging!

**Pull request merged**